src/nl/rubensten/texifyidea/modules/DefaultFileCreator.kt

```diff
@@ -0,0 +1,77 @@
+package nl.rubensten.texifyidea.modules
+
+import com.intellij.openapi.project.Project
+import nl.rubensten.texifyidea.TeXception
+import nl.rubensten.texifyidea.templates.LatexTemplatesFactory
+import java.io.File
+import java.io.FileOutputStream
+import java.io.IOException
+
+/**
+ * This class provides functionality to create default files from templates, which is useful for example when creating a new project.
+ */
+class DefaultFileCreator(
+        /** The project to add the file to. */
+        val project: Project,
+        /** The directory path of the file (no seperator and no file name). */
+        val path: String) {
```

Comment:
```suggestion
        val path: String
) {
```

I think that when documented properties inside the constructor it looks better to format it like this:
```kotlin
class DefaultFileCreator(
        
        /** 
         * The project to add the file to. 
         */
        val project: Project,
        
        /**
         * The directory path of the file (no seperator and no file name). 
         */
        val path: String
) {
```
This is my personal opinion though, don't know what you think about this?

Reply:
I don't mind, so let's just change it. I'm trying to remember your code style but it's hard.